cl_dll/view.cpp

```diff
@@ -661,6 +664,7 @@ void V_CalcNormalRefdef ( struct ref_params_s *pparams )
 	view->angles[YAW]   -= bob * 0.5;
 	view->angles[ROLL]  -= bob * 1;
 	view->angles[PITCH] -= bob * 0.3;
+	VectorCopy(view->angles, view->curstate.angles);
```

Comment:
What is this for? Seems unrelated to the change.

Reply:
It makes the viewmodel bob go corner-to-corner as shown in the video instead of just back-and-forth, I think that's why it's commented "throw in a little tilt"

Reply:
I'm asking specifically about your VectorCopy addition.

Reply:
The VectorCopy actually makes the bob go corner-to-corner (I think.)

Reply:
Could you remove it and check? I'd rather not introduce extra lines of code with little understanding of what they do.

Reply:
I already did that with [another SDK](https://github.com/thephoenixprojctsoftware/zamnhl). Removing it does not allow the bob to go corner-to-corner, the bob goes back-and-forth.

Reply:
I checked myself and it does indeed make that difference. Alright.